### PR TITLE
Adds a Callisto accent tag.

### PR DESCRIPTION
**Pull request by TheStryker**
Adds a Callisto accent tag and tweaks the existing Jovian one.
Descriptions done by Schwann.
image
image

### CODE REVIEW

**Comment by TheStryker** [comment]
!review

File: code/modules/background/accent/human.dm [hunk 1]
```diff
@@ -167,12 +167,20 @@
...
-	is typically spoken. Ganymede is often seen as the \"standard\" Jovian accent, though some on Castillo vehemently believe that the Castillian accent should be the standard."
+	description = "The Jovian accent is an odd mish-mash of accents from Jupiter’s various moons, and has heavy variation on each of the moons. On Europa, the Jovian accent often takes on traces of \
+	Tradeband due to the amount of high-income tourism, while the Castillean Jovian accent is known for the rapid, breathless pace at which it is typically spoken. Ganymede is often seen as the 'standard' \
+	Jovian accent, though some on Castillo vehemently believe that the Castillian accent should be the standard."
 	tag_icon = "jupiter"
 	text_tag = "JUP"
 
+/datum/accent/callisto
+	name = ACCENT_CALLISTO
+	description = "Considered to be one of the most recognizable accents in the Sol System due to its uniquely breathless pace, the stereotypical Callistean accent has its roots in the moon’s \
```
Comment (SleepyGemmy) [inline code/modules/background/accent/human.dm:178]:
```suggestion
	description = "Considered to be one of the most recognizable accents in the Sol System due to its uniquely breathless pace, the stereotypical Callistean accent has its roots in the moon's \
```

File: code/modules/background/accent/human.dm [hunk 2]
```diff
@@ -167,12 +167,20 @@
...
+	port district of New Valletta. While not all Callisteans speak with the breathless pace of those from New Valletta, the district’s status as a port has led to its accent becoming the moon’s most \
```
Comment (SleepyGemmy) [inline code/modules/background/accent/human.dm:179]:
```suggestion
	port district of New Valletta. While not all Callisteans speak with the breathless pace of those from New Valletta, the district's status as a port has led to its accent becoming the moon's most \
```

File: code/modules/background/accent/human.dm [hunk 3]
```diff
@@ -167,12 +167,20 @@
 
 /datum/accent/jupiter
 	name = ACCENT_JUPITER
-	description = "The Jovian accent is an odd mish-mash of accents from Jupiter's various moons, and has heavy variation on each of the moons. On Idris Incorporated-dominated Europa, \
-	the Jovian accent often takes on traces of Tradeband due to the amount of high-income tourism, while the Castillean Jovian accent is known for the rapid, breathless pace at which it \
-	is typically spoken. Ganymede is often seen as the \"standard\" Jovian accent, though some on Castillo vehemently believe that the Castillian accent should be the standard."
+	description = "The Jovian accent is an odd mish-mash of accents from Jupiter’s various moons, and has heavy variation on each of the moons. On Europa, the Jovian accent often takes on traces of \
```
Comment (SleepyGemmy) [inline code/modules/background/accent/human.dm:170]:
```suggestion
	description = "The Jovian accent is an odd mish-mash of accents from Jupiter's various moons, and has heavy variation on each of the moons. On Europa, the Jovian accent often takes on traces of \
```

File: code/modules/background/accent/human.dm [hunk 4]
```diff
@@ -167,12 +167,20 @@
...
+	Tradeband due to the amount of high-income tourism, while the Castillean Jovian accent is known for the rapid, breathless pace at which it is typically spoken. Ganymede is often seen as the 'standard' \
```
Comment (SleepyGemmy) [inline code/modules/background/accent/human.dm:171]:
```suggestion
	Tradeband due to the amount of high-income tourism, while the Castillean Jovian accent is known for the rapid, breathless pace at which it is typically spoken. Ganymede is often seen as the standard \
```

File: code/modules/background/accent/human.dm [hunk 5]
```diff
@@ -167,12 +167,20 @@
...
+	Jovian accent, though some on Castillo vehemently believe that the Castillian accent should be the standard."
```
Comment (SleepyGemmy) [inline code/modules/background/accent/human.dm:172]:
```suggestion
	Jovian accent, though some on Castillo vehemently believe that the Castillian accent should be considered the standard."
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by KingOfThePing — APPROVED** [review]

File: code/modules/background/accent/human.dm [hunk 6]
```diff
@@ -167,12 +167,20 @@
 
 /datum/accent/jupiter
 	name = ACCENT_JUPITER
-	description = "The Jovian accent is an odd mish-mash of accents from Jupiter's various moons, and has heavy variation on each of the moons. On Idris Incorporated-dominated Europa, \
-	the Jovian accent often takes on traces of Tradeband due to the amount of high-income tourism, while the Castillean Jovian accent is known for the rapid, breathless pace at which it \
-	is typically spoken. Ganymede is often seen as the \"standard\" Jovian accent, though some on Castillo vehemently believe that the Castillian accent should be the standard."
+	description = "The Jovian accent is an odd mish-mash of accents from Jupiter's various moons, and has heavy variation on each of the moons. On Europa, the Jovian accent often takes on traces of \
+	Tradeband due to the amount of high-income tourism, while the Castillean Jovian accent is known for the rapid, breathless pace at which it is typically spoken. Ganymede is often seen as the standard \
+	Jovian accent, though some on Castillo vehemently believe that the Castillian accent should be considered the standard."
```
Comment (SleepyGemmy) [inline code/modules/background/accent/human.dm:172]:
```suggestion
	Jovian accent, though some on Castillo vehemently believe that the Castillean accent should be considered the standard."
```

**Comment by Jaraci** [comment]
Please consider giving this accent to the Idris Incorporated megacorporate origin too. The Callisto wiki page already supports this from a lore perspective.

**Review by SleepyGemmy — APPROVED** [review]
suggested changes addressed.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**